### PR TITLE
Adding ELI5 video to the list of GraphQL videos

### PR DESCRIPTION
**Pull request by dmitryvinn**
## Description
Here at Meta Open Source, we've been working on creating a series of short videos about open source projects, and we recently covered GraphQL. It would be great to add it to the list on the official site.

### CODE REVIEW

**Comment by Urigo** [comment]
@dmitryvinn are you able to sign the CLA?
thanks!

**Comment by saihaj** [comment]
Thanks so much for this contribution @dmitryvinn since the PR has gone stale, I'm going to close it for now to keep things tidy. If you're still interested in landing this, please feel free to reopen a fresh pull request and sign the CLA. Thanks again!
